docs-ref-services/latest/security-keyvault-jca-readme.md

```diff
@@ -219,16 +218,16 @@ This project has adopted the [Microsoft Open Source Code of Conduct][microsoft_c
 <!-- LINKS -->
 [Source code]: https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-jca_2.6.0/sdk/keyvault/azure-security-keyvault-jca/src
 [API reference documentation]: https://azure.github.io/azure-sdk-for-java/keyvault.html#azure-security-keyvault-jca
-[Product documentation]: https://docs.microsoft.com/azure/key-vault/
+[Product documentation]: /azure/key-vault/
 [Samples]: https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-jca_2.6.0/sdk/keyvault/azure-security-keyvault-jca/src/samples/java/com/azure/security/keyvault/jca
 [azure_subscription]: https://azure.microsoft.com/
-[azure_keyvault]: https://docs.microsoft.com/azure/key-vault/keys/quick-create-portal
-[jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
+[azure_keyvault]: /azure/key-vault/keys/quick-create-portal
+[jdk_link]: /java/azure/jdk/?view=azure-java-stable&preserve-view=true
```

Comment:
I would personally remove everything after the "?" here as well.  Both https://docs.microsoft.com/en-us/azure/developer/java/fundamentals/?view=azure-java-stable and https://docs.microsoft.com/en-us/azure/developer/java/fundamentals take you to the same place.